src/services/suite.service.js

```diff
@@ -14,26 +20,23 @@ const tasksQuery = gql`query {
 `;
 
 export const SuiteService = {
-  createGraphqlClient(host, port) {
-    return createApolloClient(`http://${host}:${port}/graphql`);
+  createGraphqlClient() {
+    return createApolloClient(`${window.location.pathname}/graphql`);
```

Comment:
Here we were sending requests to whatever `cylc scan` returned for `host:port`. That's because we were sending the queries to `cylc/cylc-flow`. Instead now we copy the URL + `/graphql`, which should match the Tornado Handler for GraphQL.